scss/.sassdocrc

```diff
@@ -1,12 +1,12 @@
 # .sassdocrc (yaml)
 
-dest: './documentation/0.6.75'
+dest: './documentation/herman-test'
 descriptionPath: './README.md'
 shortcutIcon: '../sinister-logo.svg'
 herman:
   sass:
     includePaths:
-      - 'test-sin/scss'
-    use:
-      - file: 'sin'
-      - namespace: '*'
+      - 'sin-core'
+    # use:
+    #   - file: 'sin'
+    #   - namespace: '*'

```

Comment:
Anything here will be `@use`d by default in *all* `@example` annotation code. Files here should not contain any CSS output.

Reply:
I ended up using an individual `@use` in each example and it works! 

I did several tries to make `use:`  to work, but didn't manage it yet. Probably I am doing something wrong on my end, but in the end I don't mind having an `@use` inside each example.